pkg/infrared/infrared.go

```diff
@@ -45,14 +58,20 @@ type Infrared struct {
 	listeners []*Listener
 	srvs      []*Server
 	bufPool   sync.Pool
+	conns     map[net.Addr]*conn
```

Comment:
What's your reason for adding this?

Reply:
This is to keep track of all current connections. But I don't think it's really used yet. We could delete it and re-add it when we need it in the future.

Reply:
Yea, i think that would be better. Unless you think a feature which will be using this should be part of the MVP.

Reply:
I was thinking a lot about what I actually want from this version before we release it. Especially the MVP part stuck with me, but I think it's a bit too late for that since we already know that Infrared is going to be used. Rather than focusing on a MVP I would want to focus on a predefined set of features.

Reply:
> Rather than focusing on a MVP I would want to focus on a predefined set of features.

For me a MVP is a product which has a predefined set of features (but just which we label as necessary to have). So for me they're the same thing, because minimal is just a context depending word.  

But if you could make a list with features which we want to make everything more clear, lets go for it! Lets make an issue for it or something

Reply:
I created the roadmap as a Discussion. Probably the best format for it on Github https://github.com/haveachin/infrared/discussions/198